src/exceptions.md

```diff
@@ -1,33 +1,36 @@
 # Exception handling
 
-During the "Memory layout" section we decided to start out simple and leave out handling of
-exceptions. In this section we'll add support for handling exceptions; this serves as an example of
+During the "Memory layout" section, we decided to start out simple and leave out handling of
+exceptions. In this section, we'll add support for handling them; this serves as an example of
 how to achieve compile time overridable behavior in stable Rust (i.e. without relying on the
 unstable `#[linkage = "weak"]` attribute, which makes a symbol weak).
 
 ## Background information
 
 In a nutshell, *exceptions* are a mechanism the Cortex-M and other architectures provide to let
-applications respond to asynchronous, usually external, events. The exception mechanism works like
-this: when the processor receives a signal or event associated to a type of exception it suspends
+applications respond to asynchronous, usually external, events. The most prominent type of exception,
+that most people will know, is the classical (hardware) interrupt.
+
+The Cortex-M exception mechanism works like this:
```

Comment:
nit: a single line break won't render as a line break. This will render as "The Cortex-M exception mechanism works like this: When the processor receives ..." (no line break)

Reply:
I think I did not necessarily intend a line break here. Should we have one?

Reply:
Then this is fine. (I thought you wanted to insert a line break in the rendered version)